open-metadata-test/open-metadata-fvt/access-services-fvt/subject-area-fvt/src/main/java/org/odpi/openmetadata/accessservices/subjectarea/fvt/RelationshipsFVT.java

```diff
@@ -1423,4 +1427,36 @@ private ProjectScope createProjectScope(Project project, Term term) throws Inval
         System.out.println("CreatedProjectScopeRelationship " + createdProjectScope);
         return createdProjectScope;
     }
+
+    private void categoryHierarchyLinkFVT(Category parent, Category child) throws UserNotAuthorizedException, PropertyServerException, InvalidParameterException, SubjectAreaFVTCheckedException {
+        CategoryHierarchyLink categoryHierarchyLink = createCategoryHierarchyLink(parent, child);
+        String guid = categoryHierarchyLink.getGuid();
+        CategoryHierarchyLink gotCategoryHierarchyLink = subjectAreaRelationship.categoryHierarchyLink().getByGUID(this.userId, guid);
+        FVTUtils.validateLine(gotCategoryHierarchyLink);
+        System.out.println("Got CategoryHierarchyLink " + categoryHierarchyLink);
+        subjectAreaRelationship.categoryHierarchyLink().delete(this.userId, guid);
+        //FVTUtils.validateLine(gotTermCategorizationRelationship);
```

Comment:
remove

Reply:
No, There is "got" here. Remove after on 1439 line.

Reply:
I am not sure I understand - I suggest we do not add commented out lines of code.